### PR TITLE
SHARE-435 Studios details view

### PR DESCRIPTION
**Pull request by mahmoudZkhalil**
FE studio details page
add/list/remove comments and replies
list/remove projects
update studio cover
php unittest

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [x] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

### CODE REVIEW

File: migrations/2021/Version20210530193145.php [hunk 1]
```diff
@@ -20,10 +20,10 @@ public function getDescription(): string
...
+        $this->addSql('CREATE TABLE studio_program (id INT AUTO_INCREMENT NOT NULL, studio CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', activity INT NOT NULL, program CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', user CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', updated_on DATETIME DEFAULT NULL, created_on DATETIME NOT NULL, INDEX IDX_4CB3C24A4A2B07B6 (studio), UNIQUE INDEX UNIQ_4CB3C24AAC74095A (activity), INDEX IDX_4CB3C24A92ED7784 (program), INDEX IDX_4CB3C24A8D93D649 (user), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE studio_user (id INT AUTO_INCREMENT NOT NULL, studio CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', activity INT NOT NULL, user CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', role ENUM(\'admin\', \'member\'), status ENUM(\'active\', \'banned\', \'pending_request\'), updated_on DATETIME DEFAULT NULL, created_on DATETIME NOT NULL, INDEX IDX_EC686DD14A2B07B6 (studio), UNIQUE INDEX UNIQ_EC686DD1AC74095A (activity), INDEX IDX_EC686DD18D93D649 (user), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
```
Comment (HCrane) [inline migrations/2021/Version20210530193145.php]: Needs checking if allowed with production server!

File: translations/catroweb.en.yml [hunk 2]
```diff
@@ -779,4 +779,27 @@ tags:
...
\ No newline at end of file
+      title: Tutorial
+
+studio:
+  details:
+    general: 'general'
+    projects: 'projects'
+    comments: 'comments'
+    public: 'public'
+    private: 'private'
```
Comment (HCrane) [inline translations/catroweb.en.yml]: Check capitalization

File: translations/catroweb.en.yml [hunk 3]
```diff
@@ -779,4 +779,27 @@ tags:
...
+    join: 'Join'
+    leave: 'Leave'
+    showMore: 'show more'
+    showLess: 'show less'
```
Comment (HCrane) [inline translations/catroweb.en.yml]: Check capitalization

Comment (HCrane) [reply translations/catroweb.en.yml]: Already exits. Please use those.

**Review by HCrane — DISMISSED** [review]
Approved when the migration stuff is fixed.

File: translations/catroweb.en.yml (hunk 2 above)
Comment (mahmoudZkhalil) [reply translations/catroweb.en.yml]: will be capitalized through css for 'projects' and 'comments' and 'public' and 'private' have small initials

**Review by dmetzner — APPROVED** [review]

**Pull request merged**